x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/SearchableSnapshotsCanMatchOnCoordinatorIntegTests.java

```diff
@@ -234,6 +243,207 @@ public void testSearchableSnapshotShardsAreSkippedWithoutQueryingAnyNodeWhenThey
         }
     }
 
+    public void testSearchableSnapshotShardsAreSkippedBySearchShardsAPIWithoutQueryingAnyNodeWhenTheyAreOutsideOfTheQueryRange()
```

Comment:
can we add javadocs that explains what the test is doing. I can't possibly parse this super long name :) (that said, good idea to try to summarize what the test does in the method name)